src/huggingface_hub/keras_mixin.py

```diff
@@ -173,6 +201,9 @@ def save_pretrained_keras(
         task_name (`str`, *optional*):
             Name of the task the model was trained on. Available tasks
             [here](https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts).
+        multiple_model_save_args (`dict`, *optional*):
```

Comment:
This param seems very confusing, since its essentially the same thing as `model_save_kwargs`, but its just multiple...so then its slightly unclear to a user which one they should use.

Reply:
I was thinking of having this in kwargs as a dictionary and popping it in case multiple models are pushed, but not sure. 🤔